utils/vqm2blif/src/base/preprocess.cpp

```diff
@@ -33,7 +33,8 @@ int fix_netlist_connectivity_for_inout_pins(t_split_inout_pin* new_input_pin,
                                             t_port_vec* pin_node_sources,
                                             t_port_vec* pin_node_sinks           );
 
-void expand_ram_clocks(t_module* module);
+void expand_ram_clocks(t_module* module, string device);
```

Comment:
Add a comment of what these do (if you don't now, can leave it out).
If you have a comment on how to call it in the .cpp, it's better to put in the .h (or declaration here).